src/components/DataTable.js

```diff
@@ -79,18 +78,12 @@ export default ajaxCaller(class DataTable extends Component {
         div({ style: { display: 'flex', marginBottom: '1rem' } }, [
           childrenBefore && childrenBefore({ entities, columnSettings }),
           div({ style: { flexGrow: 1 } }),
-          div({ style: { display: 'flex' } }, [
-            h(SearchInput, {
-              style: { width: 300, borderRadius: '4px 0 0 4px' },
-              placeholder: 'Filter',
-              onChange: ({ target: { value } }) => this.setState({ textFilter: value }),
-              onSearch: ({ target: { value } }) => this.setState({ activeTextFilter: value }),
-              value: textFilter
-            }),
-            buttonPrimary({
-              style: { borderRadius: '0 4px 4px 0', borderLeft: 'none' },
-              onClick: () => this.setState({ activeTextFilter: textFilter })
-            }, [icon('search', { size: 18 })])
+          div({ style: { width: 300 } }, [
+            h(ConfirmedSearchInput, {
```

Comment:
A little bug I found that's not due to this PR since I could replicate it on dev but I thought would be worth noting: 
When you search and there aren't any results that match the input, the column gets auto-selected. I don't think this is a behavior that we want: 
<img width="1680" alt="Screen Shot 2019-05-06 at 12 03 51 PM" src="https://user-images.githubusercontent.com/42386483/57238372-3d720a80-6ff7-11e9-9690-846ce27fe471.png">

Reply:
Thanks, I'll grab this in another PR.